### PR TITLE
Helix board support

### PR DESCRIPTION
**Pull request by lancewilhelm**
Support for the Helix split ortholinear keyboard. Added two folders: one with a default keymap configuration as found in QMK and one for the custom keymap that I am currently using. Demonstrated OLED and RGB capability on a Sparkfun Pro Micro RP2040. 

In my keymap, I also adapted the code from the split module to flip OLED images and display different images based on the side of the split. Might be a nice feature to add into the OLED library down the road.

### CODE REVIEW

File: boards/helix/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+* Hardware Supported: Helix PCBs (probably rev2 only), Pro Micro RP2040 boards
+
+Copy the `kb.py` and `main.py` files from your preferred keymap folder into your root directory of your keyboard as detailed in the [KMK instructions](http://kmkfw.io/docs/Getting_Started/).
+
+## Microcontroller support
+
+Update this line in `kb.py` to any supported microcontroller in `kmk/quickpin/pro_micro`:
+
+```python
+from kmk.quickpin.pro_micro.boardsource_blok import pinout as pins
```
Comment (xs5871) [inline boards/helix/README.md:17]: missing code end of block "```".

**Review by xs5871 — COMMENTED** [review]
Is there a way to deduplicate some of that code? We also stopped merging personal keymaps, because there's a lot of them, they're personal preference, and usually don't add meaningfull functionality.

File: boards/helix/README.md (hunk 1 above)
Comment (lancewilhelm) [reply boards/helix/README.md]: fixed, thank you.

**Comment by lancewilhelm** [comment]
Makes sense! I remove the personal keymap and then rolled up the default keymap into the root helix directory. Added the keymap layouts to the root README.md.

**Comment by xs5871** [comment]
Please check the linter logs for necessary changes. Once they're fixed, I'd ask you to squash commits for a less "polluted" history.

File: boards/helix/main.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+keyboard.modules.append(split)
+
+# Media key support
+keyboard.extensions.append(MediaKeys())
+
+# CG Swap Module
+cg_swap = CgSwap()
+keyboard.modules.append(cg_swap)
+
+# Layer support``
```
Comment (xs5871) [inline boards/helix/main.py:33]:
```suggestion
# Layer support
```

**Review by xs5871 — CHANGES_REQUESTED** [review]
One last nit pick, then we're good to go. Don't worry about the git/github stuff, happens everyone eventually. (easiest way would have been to rebase and force push, if you're ever in a similar situation.)

**Comment by lancewilhelm** [comment]
It took me a couple of shots, but I think I figured out how to squash this time.  The ticks are now removed from `main.py`. Thanks.

**Review by xs5871 — APPROVED** [review]

**Pull request merged**